packages/shared/src/features/batchExport/types.ts

```diff
@@ -0,0 +1,73 @@
+import z from "zod";
+
+import { BatchExport } from "@prisma/client";
+
+import { singleFilter } from "../../interfaces/filters";
+import { orderBy } from "../../interfaces/orderBy";
+
+export enum BatchExportStatus {
+  QUEUED = "QUEUED",
+  PROCESSING = "PROCESSING",
+  COMPLETED = "COMPLETED",
+  FAILED = "FAILED",
+}
+
+export enum BatchExportFileFormat {
+  JSON = "JSON",
+  CSV = "CSV",
+}
+
+export enum BatchExportTableName {
+  Sessions = "sessions",
+}
+
+export const exportOptions: Record<
+  BatchExportFileFormat,
+  {
+    label: string;
+    extension: string;
+    fileType: string;
+  }
+> = {
+  CSV: { label: "CSV", extension: "csv", fileType: "text/csv" },
+  JSON: { label: "JSON", extension: "json", fileType: "application/json" },
+} as const;
+
+export const BatchExportQuerySchema = z.object({
+  tableName: z.nativeEnum(BatchExportTableName),
+  filter: z.array(singleFilter).nullable(),
```

Comment:
If we keep it thsi generic, how do we ensure, that we always have the right project_id filters in here?